thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/app.js

```diff
@@ -0,0 +1,18 @@
+// app entry point
+
```

Comment:
copy pasta from inline scripts. Ideally we want to use es6 modules import exports and not rely on globals. (todo)